pyproject.toml

```
@@ -26,9 +26,9 @@ requires-python = "~=3.9"
 dynamic = ["version"]
 
 dependencies = [
-    "DLite-Python >=0.3.22,<1,!=0.4.0,!=0.4.1",
+    "DLite-Python >=0.4.2,<1,!=0.4.0,!=0.4.1",
     "optimade[server] ~=0.25",
-    "oteapi-core ~=0.5.2",
+    "oteapi-core ~=0.6.0",
```

```suggestion
    "oteapi-core ~=0.5.2",
```
Do not yet support pydantic v2